network-core/src/server/transaction.rs

```diff
@@ -49,6 +52,9 @@ pub trait TransactionService {
         &mut self,
         ids: &[Self::TransactionId],
     ) -> Self::ProposeTransactionsFuture;
+
+    fn announce_transaction(&mut self, id: &Self::TransactionId)
```

Comment:
The documented design had the ability to announce multiple transactions at once.
@NicolasDP do you think it would never be useful to pack multiple transaction announcements in one protocol message? It is easily doable in gRPC by declaring the id field as repeatable.

Reply:
Oh.. missed that, I can send multiple transactions, that can be even easier for me